feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/grid/TimetableGrid.kt

```diff
@@ -121,11 +130,31 @@ fun TimetableGrid(
                 modifier = modifier,
             ) {
                 items(timetable.timetableItems) { timetableItem ->
-                    TimetableGridItem(
-                        timetableItem = timetableItem,
-                        onTimetableItemClick = { onTimetableItemClick(it.id) },
-                        scaleState = timetableState.scaleState,
+                    val isBookmarked = isBookmarked(timetableItem.id)
+                    val bookmarkLabel = stringResource(
+                        if (isBookmarked) {
+                            SessionsRes.string.remove_from_bookmark
+                        } else {
+                            SessionsRes.string.add_to_bookmark
+                        },
                     )
+                    val gotoTimetableDetailLabel = stringResource(SessionsRes.string.go_to_timetable_detail)
+
+                    ContextMenuProvider(
+                        items = {
+                            listOf(
+                                SimpleContextMenuItem(gotoTimetableDetailLabel) { onTimetableItemClick(timetableItem.id) },
+                                SimpleContextMenuItem(bookmarkLabel) { onBookmarkClick(timetableItem.id) },
+                            )
+                        },
+                    ) {
```

Comment:
We might consider providing all the features available directly from the detailed view here.
(such as calendar registration and asset display)